### PR TITLE
Reconfigured login.ts file into a more object orientated way similar

### PR DESCRIPTION
**Pull request by Dakimakuras**
to registration.ts

### CODE REVIEW

File: backend/src/controllers/login.ts [hunk 1]
```diff
@@ -5,34 +5,36 @@ import * as bcrypt from 'bcryptjs';
 
 import "reflect-metadata";
 
-export let tryToLogin = async (req: Request, res: Response, next: NextFunction) => {
-    let userRepo: UserRepo = new UserRepo();
-    const userName: string = req.body.username;
-    const userEmail: string = req.body.email;
-    const passWord: any = req.body.password;
-    const name: string = req.body.name;
+export class LoginObj {
```
Comment (rquinones93) [inline backend/src/controllers/login.ts]: Can you rename this to LoginObject, we shouldn't abbreviate names

File: backend/src/controllers/login.ts [hunk 2]
```diff
@@ -5,34 +5,36 @@ import * as bcrypt from 'bcryptjs';
...
 
-    console.log("Try to login");
-    const user = await userRepo.doesUserAlreadyExist(userName, userEmail);
+    async tryToLogin(req: Request, res: Response, next: NextFunction) {
+        let userRepo: UserRepo = new UserRepo();
+        const userName: string = req.body.username;
```
Comment (rquinones93) [inline backend/src/controllers/login.ts]: All be lower case?

File: backend/src/controllers/login.ts [hunk 3]
```diff
@@ -5,34 +5,36 @@ import * as bcrypt from 'bcryptjs';
...
-    let userRepo: UserRepo = new UserRepo();
-    const userName: string = req.body.username;
-    const userEmail: string = req.body.email;
-    const passWord: any = req.body.password;
-    const name: string = req.body.name;
+export class LoginObj {
 
-    console.log("Try to login");
-    const user = await userRepo.doesUserAlreadyExist(userName, userEmail);
+    async tryToLogin(req: Request, res: Response, next: NextFunction) {
```
Comment (rquinones93) [inline backend/src/controllers/login.ts]: Why not just login?

File: backend/src/controllers/login.ts (hunk 1 above)
Comment (Dakimakuras) [reply backend/src/controllers/login.ts]: Got it.

File: backend/src/controllers/login.ts (hunk 2 above)
Comment (Dakimakuras) [reply backend/src/controllers/login.ts]: Done

File: backend/src/controllers/login.ts (hunk 3 above)
Comment (Dakimakuras) [reply backend/src/controllers/login.ts]: Changed

**Pull request merged**